tests/providers/ssh/hooks/test_ssh.py

```diff
@@ -739,6 +740,21 @@ def test_openssh_private_key(self):
             session.delete(conn)
             session.commit()
 
+    def test_exec_ssh_client_command(self):
+        hook = SSHHook(
+            ssh_conn_id='ssh_default',
+            conn_timeout=30,
+            banner_timeout=100,
+        )
 
-if __name__ == '__main__':
-    unittest.main()
+        for attempt in tenacity.Retrying(stop=tenacity.stop_after_attempt(5)):
```

Comment:
This line (and isolating real SSH to this one function) Is most of the test fix, the rest is just refactoring code to make this a function on the hook, not the operator..

Reply:
could this always fail silently and do we care about that?

Reply:
I don't think so? If it doesn't succeed doesn't it raise the error?